lints/base.go

```diff
@@ -42,6 +42,17 @@ type LintInterface interface {
 	Execute(c *x509.Certificate) *LintResult
 }
 
+// An Enum to programmatically represent the source of a lint
+type LintType int
+
+const (
+	ZeroValue LintType = iota
+	CABFBaselineRequirements
```

Comment:
I'd rename this to `UnknownLintType`, except with whatever we rename `LintType` to.